.github/workflows/run-kata-coco-tests.yaml

```diff
@@ -65,6 +67,14 @@ jobs:
         timeout-minutes: 10
         run: bash tests/integration/kubernetes/gha-run.sh deploy-kata-tdx
 
+      - name: Deploy CoCo KBS
+        timeout-minutes: 10
+        run: bash tests/integration/kubernetes/gha-run.sh deploy-coco-kbs
+
+      - name: Install `kbs-client`
```

Comment:
I just realized that it will need to delete the `kbs-client` installed before leaving the job, because there is [this check](https://github.com/kata-containers/kata-containers/blob/main/tests/integration/kubernetes/confidential_kbs.sh#L125) to prevent `kbs-client` from being installed if it is already present on the system. In order words, on baremetal if we don't delete the binary then it will be installed only once. This might be a problem when we upgrade the KBS and that requires a newer version of `kbs-client`

Reply:
@wainersm yes I realized the same on the baremetal, what about having a function to delete the kbs-client? and that can be a step on the workflow

Reply:
I could not find a function that does this

Reply:
Another thing to pay attestion is about the required rust version 1.74 (see https://github.com/kata-containers/kata-containers/blob/main/versions.yaml#L207) to build `kbs-client`. It isn't going to upgrade the rust installation to match the version required, so if rust is already installed then it will simply check the version is >= 1.74. Otherwise, it will install rust. See https://github.com/kata-containers/kata-containers/blob/main/tests/integration/kubernetes/confidential_kbs.sh#L333

We plan to use a built version of kbs-client (Cc @portersrc) and I just open a issue to track that. Anyway, for the time being we are still build the client in the test environment.

Reply:
@wainersm what I notice about the rust version is that when you are building the `kbs-client` at least on the system that I test was requiring some packages that only a new version of rust is providing so I setup the rust version to 1.77 but hey maybe that is only happening on the platform that I test

Reply:
Hi @GabyCT !

> @wainersm yes I realized the same on the baremetal, what about having a function to delete the kbs-client? and that can be a step on the workflow maybe that should be a pre-requisite? maybe raise an issue to implement this?

Yes, that seems a good plan!

Reply:
> @wainersm I also notice that `jq` and `yq` needs to be installed on the system because multiple scripts will fail... as I was dealing with a fresh environment I had to installed them ... I think that the TDX CI machine already has them, but that is an observation to consider in maybe other fresh setups

yq and jq has always being complicated dependencies on CI. Our scripts will install these (mainly yq) in different paths, some paths (e.g. $GOPATH/bin) needs to be exported on PATH... a complete mess. @beraldoleal recently raised #9377 to bump the yq version. You may need to re-install yq on TDX to match the version once that PR is merged.

Reply:
@wainersm I raised a PR to deal with the kbs client uninstall https://github.com/kata-containers/kata-containers/pull/9461

Reply:
> We plan to use a built version of kbs-client (Cc @portersrc) 
I'm a little late to the this, but if you want to use that kbs-client binary (and not rebuild it) it's available:
```
oras pull ghcr.io/confidential-containers/staged-images/kbs-client:latest
$ oras pull ghcr.io/confidential-containers/staged-images/kbs-client:latest       
$ chmod +x kbs-client 
$ ./kbs-client --version
KBS client 0.1.0
```
It's the sample-only kbs-client. You already got `make uninstall` working in #9461, though, so up to you.

Reply:
Thanks for the info @portersrc . I just created the issue https://github.com/kata-containers/kata-containers/issues/9493